src/app/components/create-edit-event-page/create-edit-event-page.component.scss

```diff
@@ -8,13 +8,26 @@
 
 .event-form-container {
     @include flex-container(column);
-    padding: 25px 100px 25px 100px;
+    padding: 25px 100px;
 }
 
 .event-form-header {
-    @include flex-container(row, center);
+    @include flex-container(row, space-between);
+    align-items: center;
+    color: $yonder-blue;
+    padding: 0 100px;
+}
+
+.event-panel:hover mat-panel-description {
```

Comment:
try to write in scss mode, not css and write structures like this:
.event-panel:hover {
     mat-panel-description {
        color: $yonder-blue;
     }
     mat-panel-title {
         color: $yonder-blue;
     }
}
in the explanation below, I'll use "-->" meaning "contains"

.event-form-container --> event-form-item-wrapper --> event-form-item-container --> event-form-item

and also for  event-form-item-container --> upload-image-component-container and uploaded-image-container